docs/flink-connector/flink-catalog-jdbc.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Flink connector jdbc catalog"
+slug: /flink-connector/flink-catalog-jdbc
+keyword: flink connector jdbc catalog
+license: "This software is licensed under the Apache License version 2."
+---
+
+This document provides a comprehensive guide on configuring and using Apache Gravitino Flink connector to access the JDBC catalog managed by the Gravitino server.
+
+## Capabilities
+
+### Supported JDBC Types
+
+* MYSQL
+
+## Getting Started
+
+### Prerequisites
+
+Place the following JAR files in the lib directory of your Flink installation:
+
+- [flink-connector-jdbc-${flinkJdbcConnectorVersion}.jar](https://nightlies.apache.org/flink/flink-docs-release-1.18/docs/connectors/table/jdbc/)
+- [gravitino-flink-connector-runtime-1.18_2.12-${gravitino-version}.jar](flink-connector.md)
```

Comment:
could you change the link to `https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-flink-connector-runtime-1.18` ?